packages/redhat-rpm-config/kernel-module-symbol-requires/runtest.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   runtest.sh of /CoreOS/redhat-rpm-config/Regression/bz1619235-Incorrect-kernel-module-symbol-Requires
+#   Description: Test for BZ#1619235 (Incorrect kernel module symbol Requires generation)
+#   Author: Eva Mrakova <emrakova@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2018 Red Hat, Inc.
+#
+#   This program is free software: you can redistribute it and/or
+#   modify it under the terms of the GNU General Public License as
+#   published by the Free Software Foundation, either version 2 of
+#   the License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE.  See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program. If not, see http://www.gnu.org/licenses/.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Include Beaker environment
+. /usr/bin/rhts-environment.sh || exit 1
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+PACKAGE="redhat-rpm-config"
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlAssertRpm $PACKAGE
+        rlIsRHEL '<8' || rlAssertRpm kernel-rpm-macros
+        rlRun "TmpDir=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "cp test* $TmpDir"
+        rlRun "pushd $TmpDir"
+        rlRun "mkdir -p ~/rpmbuild/SOURCES ~/rpmbuild/SPECS"
+        rlRun "mv *.tar.gz ~/rpmbuild/SOURCES"
+        rlRun "mv *.spec ~/rpmbuild/SPECS"
+    rlPhaseEnd
+
+    rlPhaseStartTest
+        # for proper rebuild of non-x86_64 arches ARCH has to be unset
+        ARCHBACKUP=$ARCH
+        unset ARCH
+        uname -a
+        rpm -q kernel kernel-devel
+        KVER=$(rpm -q --qf '%{VERSION}' kernel-devel)
```

Comment:
Will this always work? I mean, we can't guarantee that there will be only one kernel-devel package installed on the system, can we? eg. it will be installed for the base kernel in kickstart

Reply:
@veruu it seems to pass everywhere in the brew pipeline, it grabs the latest kernel/kernel-devel. I"ll include example jobs in the ticket.

Reply:
Yes I believe it's a custom dependency for the test, I can double check with @ksrot

Reply:
But the only reason it passes is because there's only one kernel-devel package, no? What happens if we eg. onboard Stanislav's test which grabs another kernel from brew to determine kabi changes and that one needs to install the devel package of that kernel, or the default kickstart changes and installs kernel-devel of the GA?

Reply:
Hi,
we do not really have experience with building kernel modules and therefore we are trying to use the simplest method available. The only purpose is to ensure that we are building against kernel version which has kernel-devel installed. You are right that with more packages installed it won't work properly, obviously this was not the case we would be hitting in our testing (as we do not touch kernel). Fell free to change this line to whatever you find appropriate, I would add | head -n 1 to our test source. Thanks for pointing this out and reaching out to me.

Reply:
thanks for the comment! `uname -r` sounds good to me